### ISSUE TITLE
Cannot resolve module 'awesome-typescript-loader'

### ISSUE THREAD

**Issue by horte** [s-panferov/awesome-typescript-loader#14]
Firstly, thanks for a great loader.

Previously I used `typescript-loader` but since we use Azure I switched to using `awesome-typescript-loader` after windows path bugs were fixed in this project. Now it works great to build on windows though I am using OS X and when I try to build my project locally I get following error 
```
ERROR in multi main
Module not found: Error: Cannot resolve module 'awesome-typescript-loader' in /Users/Fredrik/path/to/project
 @ multi main
```

Any idea what might be wrong? I have all `node_modules` installed and even tried a clean `npm install`. Do I need to resolve the module manually in the config?

```
//webpack.config.js

var webpack = require("webpack");
var path = require('path');

module.exports = {
    entry: {
        main: ['./src/js/main']
    },

    output: {
        path: path.join(__dirname, "dist"),
        filename: "main.js",
        sourceMapFilename: "debug/[name].map"
    },

    // Source maps support (or 'inline-source-map' also works)
    devtool: 'source-map',

    resolve: {
        root: path.join(__dirname),
        // Allow to omit extensions when requiring these files
        extensions: ['', '.ts', '.webpack.js', '.web.js', '.js'],
        modulesDirectories: ['node_modules']
    },
    module: {
        loaders: [
            {
                test: /\.ts$/,
                loader: 'awesome-typescript-loader'
            }
            ,{
                test: /\.css$/,
                loader: "style-loader!css-loader" }
            ,{
                test: /\.less$/,
                loader: "style!css!less"
            }
        ]
    }

};
```

```
// package.json

  "dependencies": {
    "crossroads": "^0.12.0",
    "css-loader": "^0.12.0",
    "exports-loader": "^0.6.2",
    "git-rev": "^0.2.1",
    "gulp": "^3.6.2",
    "gulp-clean": "^0.3.1",
    "gulp-deploy-azure-cdn": "~1.0.0",
    "gulp-header": "^1.2.2",
    "gulp-preprocess": "~1.2.0",
    "gulp-rename": "~1.2.0",
    "gulp-strip-debug": "~1.0.2",
    "gulp-tsc": "^0.9.3",
    "gulp-uglify": "^1.1.0",
    "gulp-util": "~3.0.4",
    "gulp-watch": "^4.2.4",
    "gulp-webpack": "^1.3.0",
    "html-loader": "^0.3.0",
    "knockout": "^3.3.0",
    "less-loader": "^2.2.0",
    "run-sequence": "^0.3.6",
    "strip-loader": "~0.1.0",
    "style-loader": "^0.12.2",
    "superagent": "^1.2.0",
    "webpack": "^1.7.3",
    "webpack-dev-server": "^1.7.0",
    "awesome-typescript-loader": "^0.3.0"
  },
  "engines": {
    "node": "0.10.x"
  }
}
```

**Comment by s-panferov** [comment]
@horte sorry but I've accidentally published the version without /dist folder. Fixed in v0.3.1. Please try and say if something is wrong.

**Comment by horte** [comment]
Works like a charm, thx!
